sentry-rails/lib/sentry/rails/tracing/abstract_subscriber.rb

```diff
@@ -43,7 +43,7 @@ def record_on_current_span(duration:, **options)
             Sentry.with_child_span(**options) do |child_span|
               # duration in ActiveSupport is computed in millisecond
               # so we need to covert it as second before calculating the timestamp
-              child_span.set_timestamp(child_span.start_timestamp + duration / 1000)
+              child_span&.set_timestamp(child_span.start_timestamp + duration / 1000)
```

Comment:
In `sentry-rails`'s case, I think we may just remove all the subscribers (either not subscribe them or unsubscribe them, depends on the timing), instead of making them run and deal with `nil` span?

Reply:
yea, good point, I wasn't sure where to do this. Will just disable the subscribers.

Reply:
done